diff --git a/dbm/DBMAllocator.h b/dbm/DBMAllocator.h
index d2d2e50..4510a0e 100644
--- a/dbm/DBMAllocator.h
+++ b/dbm/DBMAllocator.h
@@ -29,7 +29,7 @@ namespace dbm
     {
     public:
         ///< Default constructor
-        DBMAllocator(): freeList(16), dbm1x1(1)
+        DBMAllocator(): freeList(256), dbm1x1(1)
         {
             raw_t lezero = dbm_LE_ZERO;
             dbm1x1.newCopy(&lezero, 1);
